polling_stations/apps/bug_reports/templates/bug_reports/report_form_view.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+
+{% load dc_forms %}
+{% load i18n %}
+{% load static from staticfiles %}
+
+{% block content %}
+
+<div class="row">
+  <div class="columns large-8">
```

Comment:
`<div class="row">` and `<div class="columns large-8">` isn't used in base theme 0.3. both can just be removed.

Reply:
Sorry, realised this was replaced later in the PR 👍